### PR TITLE
create dynamic cards

### PR DESCRIPTION
**Pull request by AlonaLook**
Please, check and give me your recommendations.

### CODE REVIEW

File: static/js/cards.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const getLinksCard = (cardLinks = []) => {
+	return cardLinks.map(({ title, href }) => {
+		const link = `<div class="card__text"><a href= ${href} >${title}</a></div>
```
Comment (PritulaRoman) [inline static/js/cards.js]: const link = `<div class="card__text">
  <a href="${href}">${title}</a>
</div>

File: static/js/cards.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+axios.get('https://jsonblob.com/api/1c066a98-2fd6-11e9-9080-df955f1091f2')
+  .then((response) => {
+  	const result = response.data.items;
+
+  	const cards = result.map(({ title, description, links }) => {
+  		const linksCard = getLinksCard(links);
+  		
+		const card = `	<div class="card">
+							<div class="card__item card__img card__img--api"></div>
+							<div class="card__item card__title" name="api">${title}</div>
```
Comment (PritulaRoman) [inline static/js/cards.js]: Do we need `name` attribute?

Comment (AlonaLook) [reply static/js/cards.js]: Yes, I need for my links!

File: static/js/cards.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+					`;
+		return card;
+  	}).join('');
+
+  	$('.box-cards').html(cards);
+ 
+  })
+
+  .catch((error) => {
+  	console.log(error);
```
Comment (PritulaRoman) [inline static/js/cards.js]: In addition :)

Please, insert some error HTML instead of cards in case of API failure

File: static/js/cards.js (hunk 1 above)
Comment (AlonaLook) [reply static/js/cards.js]: What does it mean? I didn't catch!

File: static/js/cards.js (hunk 3 above)
Comment (AlonaLook) [reply static/js/cards.js]: ok, thanks!

**Pull request merged**